conference.js

```diff
@@ -629,12 +645,19 @@ export default {
 
             setGUMPendingStateOnFailedTracks(tracks, APP.store.dispatch);
 
-            return this._setLocalAudioVideoStreams(tracks);
+            const promise = this._setLocalAudioVideoStreams(tracks);
+
+            promise.finally(() => {
+                logger.debug('Resolving initPromise! prejoinVisible=true');
```

Comment:
Is this print content correct? This is the case when preJoin is not visible.

Reply:
Yeah but it has been displayed.

Reply:
Oh 🤦 there are two checks isPrejoinPageVisible on inside another, I haven't noticed that.